sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Models/AnalyzeError.cs

```diff
@@ -3,11 +3,10 @@
 
 using Azure.Core;
 
-namespace Azure.AI.Language.Conversations.Models
+namespace Azure.AI.Language.Conversations
```

Comment:
I realized we've moved these so they're manually written now, but we may still want them in the `.Models` namespace.  KC & Ted own that decision.

Reply:
@KrzysztofCwalina @tg-msft and others originally wanted them in the main repo because most were largely for input to the 1 and only method that was there for GA; however, I agree that, with more models coming in soon, we may want to reconsider but must do so now because after this 1.1.0 beta ships I'm turning right around and finishing up the 1.0.0 GA.

Reply:
I started a thread internally with the archbaord as well.

Reply:
Talked with @tg-msft offline today and he feels nixing the Models sub-namespace is more likely to pass archboard, but we'll see next week with the CLU 1.0 GA review.